### PR TITLE
[jax2tf] Fix lax.div_p

### PR DESCRIPTION
**Pull request by petebu**

### CODE REVIEW

File: jax/experimental/jax2tf/jax2tf.py [hunk 1]
```diff
@@ -501,7 +501,7 @@ def _population_count(x):
 
 def _div(lhs, rhs):
   if lhs.dtype.is_integer:
-    quotient = tf.math.floor_divide(lhs, rhs)
+    quotient = tf.math.floordiv(lhs, rhs)
```
Comment (gnecula) [inline jax/experimental/jax2tf/jax2tf.py:504]: This suggests that we have no tests for _div for integer types?

@SIben was looking at expanding test coverage. FYI.

**Review by gnecula — APPROVED** [review]

File: jax/experimental/jax2tf/jax2tf.py (hunk 1 above)
Comment (SIben) [reply jax/experimental/jax2tf/jax2tf.py:504]: Yes, it is somehow the same problem that we had for `add` and `mul` – it is only included in `test_binary_elementwise` afaict, which only considers floating point types. Perhaps it'd be good to gather the untested paths by dtype to get a better overview.

**Comment by gnecula** [comment]
But aren't these tested in `test_type_promotion` for integer types? I suspect that the problem there is that the right combination that would trigger the error is not picked up.

**Comment by SIben** [comment]
In `test_type_promotion` we go through `jnp.divide` which ends up promoting the args to inexact types before they encounter the primitive.

**Pull request merged**